python/lsst/alert/packet/bin/syncAllSchemasToRegistry.py

```diff
@@ -0,0 +1,140 @@
+#!/usr/bin/env python
+#
+# This file is part of alert_packet.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+import argparse
+import json
+import re
+import fastavro
+import requests
+
+import lsst.alert.packet
+
+
+def parse_args():
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "--schema-registry-url",
+        type=str,
+        default="http://alert-schemas.localhost",
+        help="URL of a Schema Registry service",
+    )
+    parser.add_argument(
+        "--subject",
+        type=str,
+        default="alert-packet",
+        help="Schema Registry subject name to use",
+    )
+    return parser.parse_args()
+
+
+def upload_schema(registry_url, subject, schema_registry):
+    """Parse schema registry and upload all schemas.
+    """
+    for version in schema_registry.known_versions:
+        schema = schema_registry.get_by_version(version)
+        numbers = re.findall(r'\d+', version)
```

Comment:
Let's make the conversion of `major.minor` to integer id a method in `Schema` and use it here and in ap_association.

Reply:
I've added a method both to Schema and the schema registry so that you can find the ids.